examples/multi_fov_hcs_ome_zarr.py

```diff
@@ -13,7 +13,7 @@
 # %%
 # Set storage path
 
-store_path = f'{os.path.expanduser("~/")}hcs.zarr'
+store_path = f"{tempfile.gettempdir()}/hcs.zarr"
```

Comment:
Then probably also add a print statement so that it is clear where it got stored? So people can visualize it in napari etc.